README.md

```diff
@@ -130,10 +130,21 @@ It is important that you make changes to posts, sections, and markers through
 the `run` and `postEditor` API. This API allows the Mobiledoc editor to conserve
 and better understand changes being made to the post.
 
+```js
+editor.run(postEditor => {
+  const mention = postEditor.builder.createAtom("mention", "John Doe", { id: 42 });
+  // insert at current cursor position:
+  // or should the user have to grab the current position from the editor first?
+  postEditor.insert(mention);
+  // or specify a different position:
+  postEditor.insert(mention, position);
```

Comment:
I worry about `insert` here, especially since we already have a pretty heavily used `insertBefore` that uses a different signature and his a different meaning. As we get closer to implementing this part of the API I suggest pondering alternatives.